src/containers/email-form/email-form.jsx

```diff
@@ -1,35 +1,32 @@
 /* eslint-disable no-unused-vars */
-import React, { Component } from 'react';
-import EmailField from '../../components/email-field/email-field';
-import SubmitButton from '../../components/submit-button/SubmitButton';
-import './email-form.css'
+import React, { Component } from "react";
+import EmailField from "../../components/email-field/email-field";
+import SubmitButton from "../../components/submit-button/SubmitButton";
+import "./email-form.css";
 
 class EmailForm extends Component {
-   state = {
-   };
+    state = {};
 
-   continue = e => {
-      e.preventDefault();
-      this.props.nextStep();
-   };
+    continue = e => {
+        e.preventDefault();
+        const { nextStep } = this.props;
 
-   render() {
-      const values = this.props;
-      const handleChange = this.props;
+        nextStep();
+    };
 
-      return (
-         <div className="userInfo">
-            <EmailField
-               onChange={handleChange('email')}
-               defaultValue={values.email}
-            />
-            <SubmitButton
-               text="NEXT"
-               onClick={this.continue}
-            />
-         </div>
-      );
-   }
+    render() {
+        const { values, handleChange } = this.props;
+
+        return (
+            <div className="userInfo">
+                <EmailField
+                    onChange={handleChange}
+                    defaultValue={values.email}
+                />
+                <SubmitButton text="NEXT" onClick={this.continue} />
```

Comment:
Here you only send this.continue to the SubmitButton as onClick, onClick is just a prop name, not an event in this case, so in order for the onClick to happen you need to add an onClick on the SubmitButton